README.md

```diff
@@ -929,6 +930,36 @@ EXAMPLE
 
 _See code: [src\commands\sfpowerkit\org\orgcoverage.ts](https://github.com/Accenture/sfpowerkit/blob/master/src/commands/sfpowerkit/org/orgcoverage.ts)_
 
+## `sfpowerkit:org:profile:diff`
+
+Compare profiles from project again target org or between two orgs (source and target).
+
+```
+USAGE
+  $ sfdx sfpowerkit:org:profile:diff [-p <array>] [-s <string>] [-d <string>] [-u <string>] [--apiversion <string>] [--json] [--loglevel
+  trace|debug|info|warn|error|fatal|TRACE|DEBUG|INFO|WARN|ERROR|FATAL]
+
+OPTIONS
+  -d, --output=output                                                               Output folder. PRovide the output folder if comparing profiles from source
+                                                                                    org.
+  -p, --profilelist=profilelist                                                     List of profiles to compare. If not provided and no sourceusername is
```

Comment:
Is it going to be Coma Seperated ? it would be worth calling it out

Reply:
resolved